2 - ListsLogicAndIterations/Code/LogicAndInterations.cs

```diff
@@ -1,40 +1,87 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace ListsLogicAndIterations.Code
 {
 	public static class LogicAndInterations
 	{
 		public static int For_SumTwoListsAndAllValueThatAreNotFour(int[] numberArray, List<int> numberList)
 		{
-			throw new NotImplementedException();
-		}
+            int result = 0;
+            for(int i = 0; i < numberArray.Length; i++)
+            {
+                if (numberArray[i] != 4)
+                {
+                    result += numberArray[i];
+                }
+            }
+            for (int i = 0; i < numberList.Count; i++)
+            {
+                if (numberList[i] != 4)
+                {
+                    result += numberList[i];
+                }
+            }
+            return result;
```

Comment:
This works, but you can reduce the code duplication by working with the same list which makes it easier to maintain:

```
			var actual = 0;

			numberList.AddRange(numberArray);

			for (int i = 0; i < numberList.Count; i++)
			{
				var value = numberList[i];

				if (value != 4)
				{
					actual += value;
				}
			}

			return actual;
```